Mathlib/Analysis/InnerProductSpace/Projection.lean

```diff
@@ -845,6 +845,14 @@ theorem Submodule.isCompl_orthogonal_of_completeSpace [HasOrthogonalProjection K
   ⟨K.orthogonal_disjoint, codisjoint_iff.2 Submodule.sup_orthogonal_of_completeSpace⟩
 #align submodule.is_compl_orthogonal_of_complete_space Submodule.isCompl_orthogonal_of_completeSpace
 
+theorem eq_iff_orthogonalComplement_eq {L : Submodule 𝕜 E}[HasOrthogonalProjection K]
+    [HasOrthogonalProjection L] : K = L ↔ Kᗮ = Lᗮ := by
```

Comment:
I would reverse the order (which forces a rename), and then it can be a `simp` lemma. It can't be a `simp` lemma without reverse the order because then it would just keep adding `ᗮ` repeatedly.
```suggestion
@[simp]
theorem orthogonalComplement_eq_orthogonalComplement {L : Submodule 𝕜 E} [HasOrthogonalProjection K]
    [HasOrthogonalProjection L] : Kᗮ = Lᗮ ↔ K = L := by
```